models/users.js

```diff
@@ -304,6 +306,27 @@ const updateUserPicture = async (image, userId) => {
   }
 };
 
+/**
+ * Sets the monthlyHours field of passed UserId to userNumberOfHours
+ *
+ * @param monthlyHours { integer }: userNumberOfHours
+ * @param userId { string }: User id
+ */
+
+const updateMonthlyHours = async (userNumberOfHours, userId) => {
+  const userStatusDocs = await userStatusModel.where("userId", "==", userId).limit(1).get();
+  const [userStatusDoc] = userStatusDocs.docs;
+  if (userStatusDoc) {
+    const docId = userStatusDoc.id;
+    return userStatusModel.doc(docId).update({
+      monthlyHours: {
+        committed: 4 * userNumberOfHours,
+      },
+    });
+  }
+  return {};
+};
```

Comment:
this function should update the number of hours. the 4 * userNumberOfHours logic should be either in the controller or from frontend